src/Servers/Kestrel/test/Interop.FunctionalTests/HttpClientHttp2InteropTests.cs

```diff
@@ -318,7 +318,7 @@ public async Task BidirectionalStreaming(string scheme)
             await host.StopAsync().DefaultTimeout();
         }
 
-        [ConditionalTheory(Skip = "https://github.com/dotnet/corefx/issues/39404")]
```

Comment:
I know these were ConditionalTheories before, but why? I don't see the actual skip condition anywhere.

Reply:
Changed.